moja-kuhinjica-frontend/src/components/input/FormInput.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import styles from './FormInput.module.scss'
+import Image from 'next/image'
+
+interface IFormInputProps {
+    src: string
+    content: string
```

Comment:
You should name this prop `placeholder` instead of `content`. Makes more seance

Reply:
Done.